lib/class-wp-rest-styles-controller.php

```diff
@@ -9,6 +9,14 @@
  * Class WP_REST_Styles_Controller
  */
 class WP_REST_Styles_Controller extends WP_REST_Dependencies_Controller {
+
+	/**
+	 * Core assets collection.
+	 *
+	 * @var null|array
+	 */
+	protected $core_assets;
```

Comment:
Cant we just add this to the `WP_REST_Dependencies_Controller` class?

Reply:
@spacedmonkey Moved there.